sootup.java.bytecode/pom.xml

```
@@ -38,7 +38,13 @@
 			<artifactId>dex-tools</artifactId>
 			<version>v64</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.soot-oss</groupId>
+            <artifactId>sootup.jimple.parser</artifactId>
+            <version>1.1.2-SNAPSHOT</version>
```

```suggestion
            <version>${project.version}</version>
```
So it will always get the current version